source/tutorials/openmm.rst

```diff
@@ -17,4 +23,128 @@ folder of the GitHub repository. It contains:
   NanoVer simulation using a python client.
 * `openmm_neuraminidase`: A notebook that demonstrates how to set up an interactive
   OpenMM simulation from scratch with NanoVer and change the visualisations of
-  different atom selections for a protein-ligand system.
\ No newline at end of file
+  different atom selections for a protein-ligand system.
+
+XML Files
+==============================================
+
+The XML format for serializing OpenMM simulations consists of a root element ``<OpenMMSimulation>`` containing three main components:
+
+1. **Starting Structure**: Enclosed in either ``<pdbx>`` or ``<pdb>`` tags
+2. **OpenMM Serialized System**: Enclosed in the ``<System>`` tag
+3. **OpenMM Serialized Integrator**: Enclosed in the ``<Integrator>`` tag
+
+An optional fourth component, the OpenMM serialized state, may also be included.
+The format allows for efficient storage and exchange of OpenMM simulation data, including the starting structure, system configuration, and integrator settings.
+
+XML Structure
+-------------
+
+.. code-block:: xml
+
+    <OpenMMSimulation>
+        <pdbx>
+            <!-- Content of the PDBx file -->
+        </pdbx>
+        <System ...>
+            <!-- XML content of the OpenMM serialized system -->
+        </System>
+        <Integrator ...>
+            <!-- XML content of the OpenMM serialized integrator -->
+        </Integrator>
+    </OpenMMSimulation>
```

Comment:
May be worth keeping an eye on IRL2/nanover-protocol#285, as this will add the `State` field too

Reply:
Done in this commit [0043457](https://github.com/IRL2/nanover-docs/pull/115/commits/0043457c7c21028f743a19611dc6a729dd6a82c8).